app/views/pages/crayons.html.erb

```diff
@@ -1116,9 +1116,7 @@
           <p>You have just downvoted this article.</p>
         </div>
         <div class="crayons-snackbar__actions">
-          <div class="crayons-btn-group">
```

Comment:
@nickytonline Can we require using only [INVERTED](https://github.com/thepracticaldev/dev.to/blob/master/app/javascript/crayons/Button/Button.jsx#L31-L33) buttons? If yes, then we can easily drop `crayons-btn-group` from markup even when using multiple buttons.

Ghost buttons don't really need that and we should always be using only ghost buttons in snackbar.

Reply:
We can since it's in the internals of the <SnackbarItem /> component.